### PR TITLE
RefactorKata

### PR DESCRIPTION
**Pull request by jrharris620**
Used **_Resharper_** in **Visual Studio** to refactor this `Kata` so that it is clear, concise, and readable!

### CODE REVIEW

File: RefactorKata/RefactorKata/Program.cs [hunk 1]
```diff
@@ -1,41 +1,51 @@
...
+        /// </summary>
+        /// <returns></returns>
+        private static IEnumerable<Product> GetProducts()
+        {
+            using (var conn = new SqlConnection("Server=.;Database=myDataBase;User Id=myUsername;Password = myPassword;"))
             {
-                Console.WriteLine(products[i].name);
+                var cmd = conn.CreateCommand();
+                cmd.CommandText = "select * from Products";
+                /*
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: Remove commented code that has no explaination

File: RefactorKata/RefactorKata/Program.cs [hunk 2]
```diff
@@ -1,41 +1,51 @@
...
+
+        }
+
+        /// <summary>
+        /// The purpose of this new Method is so that we can reference GetProducts in the future.
+        /// </summary>
+        /// <returns></returns>
+        private static IEnumerable<Product> GetProducts()
+        {
+            using (var conn = new SqlConnection("Server=.;Database=myDataBase;User Id=myUsername;Password = myPassword;"))
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: I'd like to see the Connection String extracted out into the App.Config and referenced with ConfigurationManager

Comment (jrharris620) [reply RefactorKata/RefactorKata/Program.cs]: I would be so happy to do so, but the project does not contain app.config. Please advise on the next step.

File: RefactorKata/RefactorKata/Program.cs [hunk 3]
```diff
@@ -1,41 +1,49 @@
...
+                Console.WriteLine(product.Name);
             }
-            Conn.Dispose();
-            Console.WriteLine("Products Loaded!");
-            for (int i =0; i< products.Count; i++)
+
+        }
+
+        /// <summary>
+        /// The purpose of this new Method is so that we can reference GetProducts in the future.
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: Terminology like "new" and "old" should be avoided.  This is somewhat self defining as well.  Basically you said the purpose is to be able to use Get Products.   A more appropriate description might be ... "Fetch all products from the database"

**Pull request closed without merging**